Flat/src/ModulesLepPho.cc

```diff
@@ -35,7 +35,7 @@ void PandaAnalyzer::SimpleLeptons() {
     matchLeps.push_back(&ele);
     matchEles.push_back(&ele);
     gt->nLooseElectron++;
-    if (gt->nLooseElectron>=2) break;
```

Comment:
For SimpleLeptons we only care about the first two, so I'd prefer to keep this at 2 instead of `NLEP`, which is more relevant for `ComplicatedLeptons`. The latter is what you use for your VV analyses anyway.

Reply:
In principle one can use the same logic everywhere, no? shouldn't it be harmless to follow the same code?

Reply:
It is harmful, because NLEP is bigger than 2, which means I'm going to be looping over many leptons that my analysis does not care about. This is harmful to CPU performance.

Reply:
The problem is that matchLeps is used to clean the jets, and therefore this affects the leptonic analyses too. I was about to write an email about it.

Reply:
But you do not use `SimpleLeptons`...

Reply:
Ah, silly me. You are right, the code is either Simple or Complicated... Nevermind, then :-)